src/makielayout/lineaxis.jl

```diff
@@ -632,7 +632,7 @@ end
 
 Gets tick labels by applying `Showoff.showoff` to `values`.
 """
-get_ticklabels(::Automatic, values) = Showoff.showoff(values)
+get_ticklabels(::Automatic, values) = convert(Vector{AbstractString}, Showoff.showoff(values))
```

Comment:
hm this looks weird to me, usually you only have to do the type widening wherever you store the result, so wherever an observable is created holding the results.

Reply:
Yeah, I don't really understand it either. Issue is that `Showoff.showoff` itself converts `Vector{LaTexStrings}` to `Vector{Strings}`, e.g.
```julia
julia> using Makie.Showoff

julia> s = [ L"x" , L"y" ]
2-element Vector{LaTeXStrings.LaTeXString}:
 L"$x$"
 L"$y$"

julia> cs1 = Showoff.showoff(s)
2-element Vector{String}:
 "L\"\$x\$\""
 "L\"\$y\$\""

julia> cs2 = convert(Vector{AbstractString}, Showoff.showoff(s))
2-element Vector{AbstractString}:
 "L\"\$x\$\""
 "L\"\$y\$\""
```

What is even more weird is that in the last one the `LaTeXString`s also decayed, e.g.
```julia
julia> cs2[1] |> typeof
String
```

But without the `convert` the fix does not work.

Reply:
Well yeah showoff is not for printing a vector of LatexStrings, so it should not hit that method at all

Reply:
I might have to take a look directly, hard to understand just from reading the diff what's going on exactly